_docs/style/text-conventions/alert-messages.md

```diff
@@ -0,0 +1,290 @@
+| --- | --- |
+| Provide a name for each domain. *null* isn't a valid domain name. | You must provide a name for each domain and *null* isn't a valid name. |
+{: .hxTable .docTable}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+Avoid using only uppercase letters.
+
+Lines with excessive capitalization are hard to read. Use all uppercase letters only for words that require it, such as a keyword, a data type, or a specific table name that's displayed in all uppercase letters to a database user. See [Capitalization]({{site.baseURL}}/style/capitalization.html) for more information.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| The requested image $UUID has automatic disk resizing disabled. | THE REQUESTED IMAGE $UUID HAS AUTOMATIC DISK RESIZING DISABLED. |
+{: .hxTable .docTable}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+When possible, include a recommendation, either a potential fix or a reference to a document for more information.
+
+Messages should provide specific information about how the user should continue.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| The system is out of virtual IP addresses. Contact Support so they can allocate more virtual IP addresses. | The system is out of virtual IP addresses. |
+|The value -1.0 can't be accepted. Specify a positive integer value for the volume size. | The value -1.0 can't be accepted. |
+{: .hxTable .docTable}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+Be specific and provide as much detailed information as possible.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| The live migration of instance <br />89a5e582-d3f3-4665-ate2-03c2114f0bib<br /> to host compute2 failed. | Live migration failed. |
```

Comment:
The instance name and the hostname should appear as they do in the UI. Since they are not styled differently from the surrounding text in the UI, I am going to leave them as is. Chatted with Catherine about this.